src/components/Settings/SaveSearchItem.vue

```diff
@@ -65,6 +65,20 @@ export default {
 		};
 	},
 	computed: {
+		savedSearchUrl() {
+			// augment url for lend/filter compatibility
+			let ssUrl = this.savedSearch?.url ?? '';
+			// update url with lend/filter path
+			ssUrl = ssUrl.replace('lend', 'lend/filter');
+			// remove loanLimit of -1 which is equivalent to not being set
+			if (ssUrl.indexOf('loanLimit=-1') > -1) {
+				ssUrl = ssUrl.replace(/loanLimit=-1/, '');
+			}
+			// clear up any duplicate separators
+			ssUrl = ssUrl.replace(/\?&/, '?').replace(/&&/, '&');
```

Comment:
Just out of curiosity, what would this look like with duplicate separators?

Reply:
`loanLimit=-1` could be in different locations. Sometimes just after the `?`, other times sandwhiched between ampersands so you couldend up with separators alongside each other once the query param and it's value are removed...